### DISCUSSION TITLE
To calculate the average stiffness tensor from the ODFs

### DISCUSSION THREAD

**Discussion by Andre-calazans** [mtex-toolbox/mtex#546] [Ask Anything] [answered]
Hi

I am trying to calculate the average stiffness tensor from the ODF, but I have got the following warning
--------------------------------------------------------------------------------------------------------
>> [CVoigt_ch, CReuss_ch, CHill_ch]    = mean(C_ch,odf_ch)
Error using tensor/mean
Too many output arguments.
---------------------------------------------------------------------------------------
C_ch is 1x1 stiffness tensor from the mineral chlorite
odf_ch is 1x1 ODF from the mineral chlorite

Please, Does anyone know what can be my error?

I created the ODF from 5 pole figures using the command

>> odf_ch=calcODF(pf_ch)

Regards,
Andre

**Comment by ralfHielscher** [comment, marked as answer]
May I ask which MTEX version you are using?

Ralf.

**Reply by Andre-calazans** [reply]
Hi Ralf,
Thank you for your prompt feedback!

The mtex version is 5.1.1

**Reply by ralfHielscher** [reply]
Then you need to update to a more recent MTEX version.

As far as I remember the old syntax was
```matlab
[CVoigt_ch, CReuss_ch, CHill_ch] = calcTensor(odf_ch,C_ch)
```
Ralf.

**Reply by Andre-calazans** [reply]
Hi Ralf,

What is the latest version?

Thank you!
Andre

**Comment by ralfHielscher** [comment]
MTEX 5.4.0

********************************************************************
Ralf Hielscher                   Tel: +371-531-38556
Fakultät für Mathematik               +371-531-22200 (Sekr.)
Technische Universität Chemnitz  Fax: +371-531-22109
Reichenhainer Str. 39            E-mail:
ralf.hielscher@mathematik.tu-chemnitz.de
D-09126 Chemnitz                 http://www.tu-chemnitz.de/~rahi
********************************************************************

**Reply by Andre-calazans** [reply]
Thank you
Andre
